src/js/view/view.js

```diff
@@ -711,6 +710,10 @@ define([
                 case states.PLAYING:
                     _resizeMedia();
                     break;
+                case states.PAUSED:
+                    if (_model.get('controls')) {
+                        _captionsRenderer.resize();
+                    }
```

Comment:
@karimJWP Build is failing because a `break;` needs to be added to this case. The linter does not allow fallthroughs.

https://jenkins.jwplayer.com/job/jw7-commercial/6550/console